src/include/processor/operator/persistent/reader/csv/parallel_csv_reader.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "base_csv_reader.h"
+
+namespace kuzu {
+namespace processor {
+
+//! ParallelCSVReader is a class that reads values from a stream in parallel.
+class ParallelCSVReader final : public BaseCSVReader {
+public:
+    static const uint64_t PARALLEL_BLOCK_SIZE;
```

Comment:
Is this expected to not have a default value?

Reply:
It is defined in the `cpp` file. I mostly did this because it makes testing nicer (changing the block size doesn't cause a giant recompile). I kind of like it in the source file though. What do you think?